addons/mail_bot/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'OdooBot',
-    'version': '1.1',
+    'version': '1.2',
```

Comment:
do we really need to bump version?

Reply:
Have to do a migration for removed inherit, this is guideline for when we do a migration on a module.